File: Selenium/Project/ProjectActivity3.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.time.Duration;

public class ProjectActivity3 extends BaseTest{
    @Test
    public void firstInfoBoxtest(){

        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        WebElement firstInfobox = driver.findElement(By.xpath("(//div/h3[@class='uagb-ifb-title'])[1]"));
        Assert.assertEquals("Actionable Training",firstInfobox.getText());

    }


}

File: PythonActivities/Project_Activity3.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity3():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    time.sleep(20)
    infoBoxTitleElement = driver.find_element(By.XPATH,"(//div/h3[@class='uagb-ifb-title'])[1]").text

    assert infoBoxTitleElement == "Actionable Training"
    print(infoBoxTitleElement)

Project_Activity3()
